UnityProject/Assets/Scripts/UI/Systems/Jobs/GUI_PlayerJobs.cs

```diff
@@ -57,6 +57,8 @@ public class GUI_PlayerJobs : MonoBehaviour
 		[Tooltip("Number of seconds to wait after selecting a job. If the player does not spawn within that time the job selection re-opens.")]
 		private float waitForSpawnTimerMax = 6;
 
+		public OccupationList EmergancyOccupationList;
```

Comment:
owo wats dis

Reply:
While it's no longer needed, I still think we should leave it here incase the singeltonSO times out or something bad happens in the future that might cause the game to softlock on the lobby.
It also prevents clients from getting kicked out on the lobby so when users encounter an issue they can copy the error logs and give it to us.

Reply:
I have no idea what you have set here

Reply:
I've manually set the default occupation list there

actually there's no other list on the project except the default one.

Reply:
makes no sense to me then, if we lose one reference we can lose this one as well.

Reply:
not really, the issue isn't in the list itself but rather how we reference them. `ScriptableobjectSingalton` is a prefab that has a script ontop of it that has a timeout for fetching SOs (for some reason?) and prefabs usually break more than SOs. Having a direct reference to the SO incase the singalton manager timesout or its prefab breaks again for whatever reason makes sense especially for forks on unitystation in the future where they will create their own occupationlists and if something breaks on their end they might need a default one to revert to.

Reply:
I really don't like this, I was about to do a test merge but can we first remove this added complexity? I think it can confuse people on the why and how it works.

Reply:
(it also has a typo, btw)